cypress/integration/homepage.spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress"/>
+
+describe("home page", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should open movie page when click on the movie", () => {
```

Comment:
This test fails  https://ibb.co/Vv57QcW  The reason is that the response from the API can be different. So you need to mock API response.

Reply:
fixed